samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -889,11 +901,14 @@ If you are certain you understand the inherent security risks, you can use this
 
             west build --board *your_board* -p always -- -DCONFIG_NRF_CLOUD_PROVISION_CERTIFICATES=y -DCONFIG_NRF_CLOUD_CLIENT_ID_SRC_COMPILE_TIME=y  "-DCONFIG_NRF_CLOUD_CLIENT_ID=\`"698d4c11-0ccc-4f04-89cd-6882724e3f6f\`""
 
-Configuring Wi-Fi access point credentials
-==========================================
+
```

Comment:
Interesting, I could have sworn they worked without the backslashes. I will fix that.

Reply:
I think it is actually because those blocks are using `code-block::` instead of `parsed-literal::`

I can switch to parsed-literal, but we will lose the powershell-vs-bash code highlighting

Reply:
Indeed, it was the wrong directive. Works now. I don't think we lost anything.